integration/docker/Dockerfile-dev

```diff
@@ -0,0 +1,112 @@
+#
+# The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+# (the "License"). You may not use this work except in compliance with the License, which is
+# available at www.apache.org/licenses/LICENSE-2.0
+#
+# This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied, as more fully set forth in the License.
+#
+# See the NOTICE file distributed with this work for information regarding copyright ownership.
+#
+
+# Use CentOS as base image for alluxio-dev image
+FROM centos:7
+
+# Note that downloads for *-SNAPSHOT tarballs are not available
+ARG ALLUXIO_TARBALL=http://downloads.alluxio.io/downloads/files/2.7.0-SNAPSHOT/alluxio-2.7.0-SNAPSHOT-bin.tar.gz
```

Comment:
will the version string here will be updated by `./dev/scripts/update-versions.sh`?

Reply:
Fixed